js/ui/overlay/ui.overlay.js

```diff
@@ -859,6 +858,8 @@ const Overlay = Widget.inherit({
             transclude,
             onRendered: () => {
                 whenContentRendered.resolve();
+
+                // NOTE: T1114344, T1130045, T390360, T386038
```

Comment:
Are really all these tickets related to this row of code?

Reply:
Move notes: [1](https://github.com/DevExpress/DevExtreme/blob/180d1d8287ef5eda0f716edd1d065fd326239a89/js/ui/popup/ui.popup.js#L428), [2](https://github.com/DevExpress/DevExtreme/blob/180d1d8287ef5eda0f716edd1d065fd326239a89/js/ui/popup/ui.popup.js#L352), [3](https://github.com/DevExpress/DevExtreme/blob/180d1d8287ef5eda0f716edd1d065fd326239a89/js/ui/overlay/ui.overlay.js#L502), [4](https://github.com/DevExpress/DevExtreme/blob/180d1d8287ef5eda0f716edd1d065fd326239a89/js/ui/overlay/ui.overlay.js#L863).